src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -215,7 +217,7 @@ const PIAFormPage = () => {
         });
       } else if (buttonValue === 'cancel') {
         if (pia?.id) {
-          navigate(`/pia/intake/${pia.id}/${pia.title}`);
+          navigate(buildDynamicPath(routes.PIA_VIEW, { id: pia.id }));
```

Comment:
will this replace seeing the title of the PIA in the link? Do we want the title to show in the links?

Reply:
It will. Adding it back.

Reply:
updated